.pre-commit-config.yaml

```diff
@@ -60,6 +60,9 @@ repos:
     rev: 1.0.5
     hooks:
       - id: shell-lint
+        # TODO: SC2039 is retired in 0.7.2 but not 0.7.0.  Local checks will
+        # pass but they will currently fail in GitHub Actions which uses 0.7.0
+        args: [-x, --exclude=SC2039]
```

Comment:
Can you create an issue to remove this when possible and include a link to it here?  One of the checkboxes in the standard PR template references doing this, so it is required.

Reply:
At @mcdonnnj 's suggestion I've implemented this differently so that it won't require a disable or a TODO.

Reply:
Some additional context about `SC2039`: 
https://github.com/koalaman/shellcheck/wiki/SC2039

And an issue about `SC30xx` codes not being properly reported:
https://github.com/koalaman/shellcheck/issues/2052

Reply:
@felddy - normally we let the reporter resolve comments.